backend-api/src/functions/asyncCredential/sessionService/sessionService.test.ts

```diff
@@ -109,17 +109,17 @@ describe("Session Service", () => {
         const dbMock = mockClient(DynamoDBClient);
         dbMock.on(GetItemCommand).rejects("Mock DB Error");
 
-        const result = await service.createSession({
-          sessionId: "137d5a4b-3046-456d-986a-147e0469cf62",
-          state: "mockValidState",
-          sub: "mockSub",
-          clientId: "mockClientId",
-          govukSigninJourneyId: "mockJourneyId",
-          redirectUri: "https://mockRedirectUri.com",
-          issuer: "mockIssuer",
-          sessionState: "mockSessionState",
-          issuedOn: "mockIssuedOn",
-        });
+        const result = await service.createSession(
+          "137d5a4b-3046-456d-986a-147e0469cf62",
```

Comment:
Similar here - can we calculate this outside of the lambda?

Reply:
Updated